File: Selenium/Activities/Activity4.py
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as chromeService
from webdriver_manager.chrome import ChromeDriverManager
from selenium.webdriver.common.by import By
service=chromeService(ChromeDriverManager().install())
with webdriver.Chrome(service=service) as driver:
    driver.get("https://v1.training-support.net/selenium/target-practice")
    print("Webpage title : ",driver.title)
    text=driver.find_element(By.XPATH,"//h3[@id='third-header']").text
    print("3rd hearder text: ",text)
    colour=driver.find_element(By.XPATH,"//h5[text()='Fifth header']").value_of_css_property("colour")
    print("Colour of 5th header: ",colour)
    violet_class=driver.find_element(By.XPATH,"//button[text()='Violet']").get_attribute("class")
    print("classes of violet button : ",violet_class)
    greyColourText=driver.find_element(By.XPATH,"//button[@class='ui grey button']").text
    print("Grey colour button text : ",greyColourText)
    driver.close()

File: Selenium/Activities/Activity4.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity4 {
    public static void main(String[]args){

        WebDriverManager.chromedriver().setup();
        WebDriver driver=new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/target-practice");
        System.out.println("Webpage title : "+driver.getTitle());
        String text=driver.findElement(By.xpath("//h3[@id='third-header']")).getText();
        System.out.println("3rd hearder text: "+text);
        String colour=driver.findElement(By.xpath("//h5[text()='Fifth header']")).getCssValue("color");
        System.out.println("Colour of 5th header: "+colour);
        String violet_class=driver.findElement(By.xpath("//button[text()='Violet']")).getAttribute("class");
        System.out.println("classes of violet button : "+violet_class);
        String greyColourText=driver.findElement(By.xpath("//button[@class='ui grey button']")).getText();
        System.out.println("Grey colour button text : "+greyColourText);
        driver.close();
    }

}
